src/Microsoft.Health.Fhir.Liquid.Converter/Filters/SegmentFilters.cs

```diff
@@ -128,5 +128,47 @@ public static bool HasSegments(Hl7v2Data hl7v2Data, string segmentIdContent)
 
             return result;
         }
+
```

Comment:
is it necessary to use segment class here?

Reply:
The advantage of using segment class over string here is that users can also refine the starting point of segmentation, and further customize the segmentList that meets their slicing intention according to their needs, not just through meta of segment.